### PR TITLE
Rips out the --binDir and --descDir and replaces them with --dir and …

### PR DESCRIPTION
**Pull request by dbernstein**
…--binaries.

Resolves https://jira.duraspace.org/browse/FCREPO-2226

### CODE REVIEW

**Comment by whikloj** [comment]
@dbernstein I'm new here, but I don't see where you are checking the includeBinaries flag?

File: src/main/java/org/fcrepo/importexport/ArgParser.java [hunk 1]
```diff
@@ -101,20 +101,20 @@ public ArgParser() {
...
-                .desc("Directory where the RDF descriptions are stored")
+                .longOpt("dir")
+                .hasArg(true).numberOfArgs(1).argName("dir")
+                .desc("The directory to export repo to or import the repo from.")
                 .required(true).build());
 
+
+        // Import/export binaries option
+        configOptions.addOption(Option.builder("b")
+                .longOpt("binary")
```
Comment (awoods) [inline src/main/java/org/fcrepo/importexport/ArgParser.java]: Please change from `binary` to `binaries`.

File: src/main/java/org/fcrepo/importexport/ArgParser.java [hunk 2]
```diff
@@ -101,20 +101,20 @@ public ArgParser() {
                 .required(false).build());
 
         // Binary Directory option
```
Comment (awoods) [inline src/main/java/org/fcrepo/importexport/ArgParser.java]: This comment should read `// Base Directory option`.

File: src/test/java/org/fcrepo/importexport/exporter/ExporterTest.java [hunk 3]
```diff
@@ -65,7 +65,7 @@
     private Config args;
     private Config binaryArgs;
     private Config metadataArgs;
-
+    private String exportDirectory = "target/export";
```
Comment (awoods) [inline src/test/java/org/fcrepo/importexport/exporter/ExporterTest.java]: This is very minor, but for readability, please add an empty line before `@Before`.

File: src/test/java/org/fcrepo/importexport/importer/ImporterTest.java [hunk 4]
```diff
@@ -56,36 +56,34 @@
     private URI pairtreeURI;
     private URI finalContainerURI;
     private File binaryFilesDir;
-
     private FcrepoResponse conResponse;
     private PutBuilder binBuilder;
-
```
Comment (awoods) [inline src/test/java/org/fcrepo/importexport/importer/ImporterTest.java]: This is very minor, but for readability, please add an empty line before `@Before`.

**Comment by awoods** [comment]
@ruebot and @whikloj, both of your comments are worthy of "ReOpening" the JIRA ticket. Please feel free to do that in the future so others know that the ticket is not ready yet.

**Comment by jhsimpson** [comment]
@ruebot, sure I will look at the docs and update accordingly.

**Pull request merged**